app/util/Youtube.scala

```diff
@@ -20,6 +20,7 @@ class YouTubeConfig @Inject()(config: Configuration) {
   lazy val clientSecret = config.getString("youtube.clientSecret").getOrElse("")
   lazy val refreshToken = config.getString("youtube.refreshToken").getOrElse("")
   lazy val contentOwner = config.getString("youtube.contentOwner").getOrElse("")
+  lazy val allowedChannels = config.getStringList("youtube.allowedChannels")
```

Comment:
I like using a name that describes what it does... especially as they're not treated any differently than other channels. `devChannels` might imply other dev features?